.github/workflows/rules.yml

```diff
@@ -41,12 +45,16 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install pytest-cov
-        pip install .[${{inputs.pip-extras}}]
-        if [[ "${{ inputs.environment }}" == "qibo" ]]; then
+        if [ -z ${{ inputs.pip-extras }} ]; then
+          pip install .
+        else
+          pip install .[${{ inputs.pip-extras }}]
+        fi
+        if [[ "${{ inputs.environment }}" == "qibojit+tensorflow" ]]; then
           pip install git+https://github.com/qiboteam/qibojit
           pip install tensorflow
         fi
-        if [[ "${{ inputs.environment }}" == "qibolab" ]]; then
+        if [[ "${{ inputs.environment }}" == "qibo" ]]; then
```

Comment:
This I believe was correct as it was:
- the meaning of "environment" was the name of the project of the caller
- if it is `qibo`, then `qibo` is already installed from the checked out version, and so from Git: what you need is just the Git version of `qibojit` (and for some reason to reinstall tensorflow afterwards)
- if it is `qibolab`, then it requires `qibo` to be installed from Git

However, it is just a terminology misunderstanding. My hope is to drop `inputs.environment` altogether, as soon as possible, using standard sections of configuration files for everything.

Reply:
Yes, I understand, but if we follow these names then qibojit will have an `environment: qibolab` which looks odd.

Reply:
Or a long if condition for qibojit and qibolab.

Reply:
Indeed, the ugly and consistent way would have been to extend the condition.

The good solution is getting rid of  `inputs.environment` altogether.

Reply:
Fine by me, reverted to OR condition.